### PR TITLE
STIJ-2: Added mobile menu molecule.

### PR DESCRIPTION
**Pull request by meirege**
- Added mobile menu molecule.
- Added mobile menu to header organism.
- Added new preview layout to preview mobile-menu inside the style guide.

### CODE REVIEW

File: components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+.mobile-menu {
+  position: fixed;
+  top: 0;
+  left: -$mobile-menu-width;
+  width: $mobile-menu-width;
+  height: 100%;
+  transition: left.5s cubic-bezier(0, 1, .5, 1);
+  background-color: $color-darkest-blue;
+  box-shadow: 0 1px 16.5px 0 rgba(0, 0, 0, .5);
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss]: - Shadow is visible when closed.
- Overlay is missing.

File: components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+  &.js-opened {
+    left: 0;
+  }
+
+  ul {
+    margin: 0;
+  }
+
+  a {
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss]: Change px into rems

File: components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
+$mobile-menu-width: 375px;
+
+.mobile-menu {
+  position: fixed;
+  top: 0;
+  left: -$mobile-menu-width;
+  width: $mobile-menu-width;
+  height: 100%;
+  transition: left.5s cubic-bezier(0, 1, .5, 1);
+  background-color: $color-darkest-blue;
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss]: Link it onto the section wrapper or move these settings to _vars.scss

File: components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  }
+
+  a.close {
+    border: 0;
+    background-color: transparent;
+    text-align: right;
+    text-decoration: none;
+
+    i {
+      padding-left: 23px;
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss]: rems

File: components/41-organisms/header/header.twig [hunk 5]
```diff
@@ -4,7 +4,7 @@
       {% include '../../21-atoms/logo/logo.twig' %}
     </div>
     <div id="example-component">
```
Comment (Jeroen005) [inline components/41-organisms/header/header.twig]: Menu should be left from the logo.

File: components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss [hunk 6]
```diff
@@ -0,0 +1,54 @@
+$mobile-menu-width: 375px;
+
+.mobile-menu {
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss]: Add media query, only on mobile.

File: components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss [hunk 7]
```diff
@@ -0,0 +1,54 @@
+$mobile-menu-width: 375px;
```
[the remark this answers is not in the document]
Comment (meirege) [reply components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss]: You are right, I just based myself on the width specified in the design... :)

File: components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss (hunk 6 above)
Comment (meirege) [reply components/31-molecules/building-blocks/mobile-menu/_mobile-menu.scss]: The component shouldn't specify this imo.

File: components/31-molecules/building-blocks/mobile-menu/mobile-menu.js [hunk 8]
```diff
@@ -1,32 +0,0 @@
-(function ($) {
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/mobile-menu/mobile-menu.js]: This file should not be removed.

File: public/styleguide/js/mobile-menu.js [hunk 9]
```diff
@@ -0,0 +1,35 @@
+(function ($) {
```
Comment (Jeroen005) [inline public/styleguide/js/mobile-menu.js]: File should be removed. ;-)

File: components/31-molecules/building-blocks/hamburger-menu/_hamburger-menu.scss [hunk 10]
```diff
@@ -0,0 +1,62 @@
+.hamburger-menu {
+  position: fixed;
+  top: 0;
+  left: -$hamburger-menu-width;
+  width: $hamburger-menu-width;
+  height: 100%;
+  transition: $hamburger-menu-transition;
+  background-color: $color-darkest-blue;
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/hamburger-menu/_hamburger-menu.scss]: Also add those colors to _vars.scss

File: components/31-molecules/building-blocks/hamburger-menu/_hamburger-menu.scss [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+    padding-left: .8rem;
+    border-bottom: 1.5px solid darken($color-darkest-blue, 2);
+    color: $color-white;
+    font-size: .7rem;
+    font-weight: bold;
+    line-height: 3.48;
+
+    &.active,
+    &:active {
+      border-color: $color-dark-blue;
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/hamburger-menu/_hamburger-menu.scss]: Also add those colors to _vars.scss

File: components/31-molecules/building-blocks/hamburger-menu/_hamburger-menu.scss [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+
+  ul {
+    margin: 0;
+  }
+
+  a {
+    display: block;
+    padding-right: .8rem;
+    padding-left: .8rem;
+    border-bottom: 1.5px solid darken($color-darkest-blue, 2);
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/hamburger-menu/_hamburger-menu.scss]: Also add those colors to _vars.scss

File: components/31-molecules/building-blocks/hamburger-menu/_hamburger-menu.scss [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+    line-height: 3.48;
+
+    &.active,
+    &:active {
+      border-color: $color-dark-blue;
+      background-color: $color-dark-blue;
+    }
+
+    &:hover {
+      border-color: darken($color-darkest-blue, 2);
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/hamburger-menu/_hamburger-menu.scss]: Also add those colors to _vars.scss

File: public/styleguide/js/mobile-menu.js (hunk 9 above)
Comment (meirege) [reply public/styleguide/js/mobile-menu.js]: The public folder and files in it are removed from git in #33

**Pull request merged**